securedrop_client/api_jobs/uploads.py

```diff
@@ -106,7 +106,7 @@ def __init__(self, message: str, reply_uuid: str):
         self.reply_uuid = reply_uuid
 
 
-class SendReplyJobTimeoutError(RequestTimeoutError):
+class SendReplyJobTimeoutError(Exception):
```

Comment:
since the exception in #820 `UnboundLocalError`, I would expect that the exception type that the job should raise should be `SendReplyJobError` - what am I missing?